lib/screens/settings/stores/settings_store.dart

```diff
@@ -144,6 +145,14 @@ abstract class _SettingsStoreBase with Store {
   // Sleep defaults
   static const defaultChatOnlyPreventSleep = false;
 
+  // mute words defaults
+  // This static function is required to create a new instance of ObservableList for the JSON serialization
+  static ObservableList<String> getDefaultMutedWords() {
+    return ObservableList<String>();
+  }
```

Comment:
For simplicity, I think we should just make `defaultMutedWords` a normal string array (follow up in next comment):
```suggestion
static const defaultMutedWords = <String>[];
```